### ISSUE TITLE
Build doesn't depend on images

### ISSUE THREAD

**Issue by azonenberg** [azonenberg/scopehal-docs#19]
If a graphic is updated, "make doc" won't rebuild the PDF until a .tex file is changed.

**Comment by kench** [comment]
I have limited knowledge of CMake, but is it because the images are not listed as a dependency and CMake won't rebuild the PDF unless one of the dependencies it tracks changes?

https://github.com/azonenberg/scopehal-docs/blob/master/CMakeLists.txt

**Comment by azonenberg** [comment]
Yes, that is correct. The challenge is to find a method that works on both Windows and Linux, and doesn't require explicitly listing every image resource in the CMake script.
